lib/google_crawler_web/templates/keyword/index.html.eex

```diff
@@ -14,59 +14,78 @@
 </section>
 <section class="row mt-3">
   <div class="col-12">
-    <div class="collapse" id="keywordFilter">
+    <div class="collapse keyword-filter" id="keywordFilter">
```

Comment:
How about moving this into a `_filter` partial @hanam1ni 🤔

Reply:
the partial refactoring is in this PR https://github.com/hanam1ni/google-crawler/pull/24 😁